src/core/lib/security/authorization/mock_cel/activation.h

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_LIB_SECURITY_AUTHORIZATION_MOCK_CEL_ACTIVATION_H
+#define GRPC_CORE_LIB_SECURITY_AUTHORIZATION_MOCK_CEL_ACTIVATION_H
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/security/authorization/mock_cel/cel_value.h"
+
+namespace google {
+namespace api {
+namespace expr {
+namespace runtime {
+
+// Base class for an activation.
```

Comment:
Add a comment that

```
// Base class for an activation. This is a temporary stub implementation of CEL APIs.
// Once gRPC imports the CEL library, this class will be removed.
```

Similarly for Activation and CelValue class.

Reply:
Thank you for the review. I have fixed this and the above issue.